### PR TITLE
Get bases by organisation id

### PR DESCRIPTION
**Pull request by DurkoMatko**
add: get bases by org id + autoformat changes

(sibling PR to FE https://github.com/boxwise/boxwise-react/pull/12)

### CODE REVIEW

File: boxwise_flask/models.py [hunk 1]
```diff
@@ -26,30 +27,34 @@ def __unicode__(self):
...
     def __unicode__(self):
         return self.name
 
     @staticmethod
     def get_camp_id(usergroup_id):
-        return (Cms_Usergroups_Camps
-            .select(Cms_Usergroups_Camps.camp_id)
-            .where(Cms_Usergroups_Camps.cms_usergroups_id == usergroup_id)
-            )
+        return Cms_Usergroups_Camps.select(Cms_Usergroups_Camps.camp_id).where(
```
Comment (DurkoMatko) [inline boxwise_flask/models.py:49]: this was caused by running `pre-commit run --all-files`

File: boxwise_flask/resolvers.py [hunk 2]
```diff
@@ -17,6 +17,12 @@ def resolve_all_camps(_, info):
     return list(response.dicts())
 
 
+@query.field("orgBases")
```
Comment (mcgnly) [inline boxwise_flask/resolvers.py:20]: how do you determine who has a right to see this info? is it considered public? common to anyone who works at any of the bases owned by the org? only accessable to people of a certain level at that org? we're going to start having to think about authorization concepts as we progress

Comment (DurkoMatko) [reply boxwise_flask/resolvers.py:20]: TBH I didn't think about it twice. Also I actually can't answer your questions who's allowed to see this and how the general authorization should be done. I just followed how "allBases" is being done and done the same

**Comment by aerinsol** [comment]
@DurkoMatko  if you can let me know when you add the comment about the auth bit, I'll merge this directly into master.

**Comment by DurkoMatko** [comment]
@aerinsol I'm not sure what my comment should be tho. I guess I'll be authorizing myself the same way we do authorize ourselves for all other data? Haven't seen any comment about this topic anywhere in the repo.

**Comment by aerinsol** [comment]
@mcgnly you've been doing some documentation on this, can you share the reference with @DurkoMatko ?

**Comment by DurkoMatko** [comment]
[Create FE header bar and link it with PHP app #small #react](https://trello.com/c/zbEEmGVo/537-create-fe-header-bar-and-link-it-with-php-app-small-react)

**Comment by aerinsol** [comment]
@DurkoMatko Alright, I spoke to @mcgnly and I understand where the question is. For a user to be able to get a list of all organisation bases, they need to do the following:

1. Be an active (non-expired) member of the organization;
2. Be a member of a usergroup that has access to that base (`cms_usergroups_camps` in the DB)
3. UNLESS they are a head of operations (highest level in `cms_usergroups_level`), in which case they can always see all of the bases.

**Pull request merged**